app/Http/Controllers/ShowDocumentationController.php

```diff
@@ -192,7 +192,7 @@ protected function extractTableOfContents(string $document): array
                 return [
                     'level' => strlen(trim(Str::before($line, '# '))) + 1,
                     'title' => $title = trim(Str::after($line, '# ')),
-                    'anchor' => Str::slug($title),
+                    'anchor' => Str::slug(Str::replace('`', 'code', $title)),
```

Comment:
does it need to have the word `code` in there? could we just replace the backtick?

Reply:
This function tries to guess the anchor generated by `\App\Support\CommonMark\HeadingRenderer`. It uses the markdown version and find lines starting with "#" and use `Str::slug` on the content. 

The problem is `HeadingRenderer` converts the content of the heading into html before doing an `Str::slug`, so `<code>MenuBarContextMenuOpened</code>` becomes `codemenubarcontextmenuopenedcode `

The proper way would be to extract the ToC from the converted HTML instead of the markdown then use an HTML parser to find headings and their anchors. Or to generate the id without `$childRenderer->renderNodes()`.

My fix isn't perfect, nor is this Markdown renderer. 

I only wanted to push a quick and easy fix.

Also, I usually use these two included renderers/extensions:
-  https://commonmark.thephpleague.com/2.5/extensions/heading-permalinks/
-  https://commonmark.thephpleague.com/2.5/extensions/table-of-contents/

Reply:
Ok, i'll merge this for now, but I'd like us to move towards the CommonMark extensions instead as that feels better than rolling our own solution to this